pkg/spec/types.go

```diff
@@ -211,42 +223,44 @@ type ControllerFields struct {
 
 	// k8s: io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
 	meta_v1.ObjectMeta `json:",inline"`
-	PodSpecMod         `json:",inline"`
-}
-
-type Controller struct {
-	Controller string `json:"controller,omitempty"`
 }
 
-// DeploymentSpecMod is Kedge's extension of Kubernetes DeploymentSpec and allows
-// defining a complete kedge application
+// DeploymentSpecMod is Kedge's extension of Kubernetes DeploymentSpec
 // kedgeSpec: io.kedge.DeploymentSpecMod
 type DeploymentSpecMod struct {
-	ControllerFields `json:",inline"`
+	// k8s: io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
+	meta_v1.ObjectMeta `json:",inline"`
+
 	// k8s: io.k8s.kubernetes.pkg.apis.apps.v1beta1.DeploymentSpec
 	ext_v1beta1.DeploymentSpec `json:",inline"`
+
+	PodSpecMod `json:",inline"`
 }
 
-// JobSpecMod is Kedge's extension of Kubernetes JobSpec and allows
-// defining a complete kedge application
+// JobSpecMod is Kedge's extension of Kubernetes JobSpec
 // kedgeSpec: io.kedge.JobSpecMod
 type JobSpecMod struct {
-	ControllerFields `json:",inline"`
+	// k8s: io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta
+	meta_v1.ObjectMeta `json:",inline88891"`
```

Comment:
/me confused here between `inline` & `inline88891`

Reply:
ah ;-) that is typo :-D (psst yubi key :innocent: ;-) )

good catch